frontend/src/pages/admin/NotificationsHub.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+    <div class="clear-page-gutters">
+        <div class="ff-instance-header">
+            <ff-page-header title="Notifications Hub" />
+        </div>
+        <div class="px-3 py-3 md:px-6 md:py-6">
+            <form class="flex flex-col gap-5" data-el="notification-form" @submit.prevent>
+                <section class="flex gap-10">
+                    <section>
+                        <FormRow v-model="form.title" type="input" placeholder="Title" class="mb-5" data-el="notification-title">
+                            Announcement Title
+                            <template #description>Enter a concise title for your announcement.</template>
+                        </FormRow>
+                        <FormRow v-model="form.message" class="mb-5" data-el="notification-message">
+                            Announcement Text
+                            <template #description>Provide the details of your announcement.</template>
+                            <template #input><textarea v-model="form.message" class="w-full max-h-80 min-h-40" rows="4" /></template>
+                        </FormRow>
+                        <FormRow v-model="form.url" type="input" :placeholder="urlPlaceholder" class="mb-5" data-el="notification-external-url">
+                            URL Link
+                            <template #description>Provide an url where users will be redirected when they click on the notification.</template>
+                        </FormRow>
+                        <ff-checkbox v-model="form.externalUrl" data-el="notification-external-url-toggle">
```

Comment:
Not sure how this is meant to be used for non-external links. It requires quite a deep knowledge of the UI - something self-hosted users won't have. Is there a specific use-case that requires this capability?

Reply:
It was mainly for us to know how to properly format the internal url's. I agree there's no use on having it for self-hosted customers. Would leaving the external url as default and hiding the placeholder for the internal routes suffice?

Reply:
The default should be external URL, and I don't think we should have the option for it to be anything else. If we wanted to add support for routing internally, then we can sniff the value to decide if its a url or an object - no need to store that otherwise.

Reply:
I removed the external url selector, forcing all urls to be external